static/app/views/issueList/overviewWrapper.tsx

```diff
@@ -1,20 +1,16 @@
 import {TAXONOMY_DEFAULT_QUERY} from 'sentry/constants';
 import {t} from 'sentry/locale';
-import type {RouteComponentProps} from 'sentry/types/legacyReactRouter';
 import {defined} from 'sentry/utils';
+import {useLocation} from 'sentry/utils/useLocation';
 import useOrganization from 'sentry/utils/useOrganization';
 import IssueListContainer from 'sentry/views/issueList';
 import IssueListOverview from 'sentry/views/issueList/overview';
 
 import {DEFAULT_QUERY} from './utils';
 
-type OverviewWrapperProps = RouteComponentProps<
-  Record<PropertyKey, string | undefined>,
-  {searchId?: string}
->;
-
-export function OverviewWrapper(props: OverviewWrapperProps) {
-  const shouldFetchOnMount = !defined(props.location.query.new);
+export function OverviewWrapper() {
+  const location = useLocation();
+  const shouldFetchOnMount = !defined(location.query.new);
```

Comment:
is it actually easier to use nuqs for this type of stuff now?

Reply:
holding off on nuqs for now until the testing story is a little further